Tests/BugsnagConfigurationTests.m

```diff
@@ -3,8 +3,9 @@
  */
 
 #import <XCTest/XCTest.h>
-#import "BugsnagTestConstants.h"
+#import <objc/runtime.h>
```

Comment:
nit: I don't believe runtime.h is required?

Reply:
It's not and this is a relic from an earlier approach - I've removed the import and rebased to resolve the previous conflicts.